contracts/Vault.sol

```diff
@@ -186,7 +225,11 @@ contract Vault is
         public
         view
         override(IVault)
-        returns (uint256)
+        returns (
+            uint256,
+            uint256,
+            uint256
+        )
```

Comment:
can we please name these return fields? (either here, on in the function docs)
It's hard to understand what each one means

Reply:
`@return amount of yield for @param _to, share of yield, and performance fee from yield`, updated description in IVault.sol